examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,13 +1183,74 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+void start_versioned_API_examples() {
```

Comment:
This looks like a good example of the API for constructing and using a `mongocxx::server_api`.

I think this needs to be updated to match the ticket description.

```
// Start Versioned API Example 1
--
 
client = MongoClient(
<connection string>,
api=\{"version": "1"}
)
 
// End Versioned API Example 1
```

Should correspond to the C++ example with those comment delimiters, something like:

```
// Start Versioned API Example 1
auto server_api = mongocxx::options::server_api(mongocxx::options::server_api::version::k_version_1);
auto client_opts = mongocxx::options::client ();
opts.server_api_opts (api_opts);
auto client = mongocxx::client(uri, client_opts);
// End Versioned API Example 1
```

The examples are used in language examples for the MongoDB documentation pages ([example, the language selector on the right](https://docs.mongodb.com/manual/changeStreams/)). So they need to closely conform to the pseudo code.